src/Controller/MenuController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\DoStuff;
+use App\Util\Debugger;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use App\Service\CategorieService;
+use App\Entity\Produit;
+use App\Entity\Menu;
+
+class MenuController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        /** @var Debugger $debugger */
+        $debugger = $container->get('debug');
+
+        $entityManager = $container->get('entity.manager');
+        $entityManager->resetDatabase();
+
+        ob_start();
+        $menu = new Menu();
+
+        echo "<h1>Créer un menu</h1> <br>";
+        echo "Prix du menu : ".$menu->getPrix()." €<br><br>";
+
+        echo "<form method=\"post\" action=\"/\">";
+
+        echo "1. Choisir un plat<br>";
+        
+        $p1 = new Produit();
+        $p1->setNom("Mon Produit");
+        $p1->setPrix(10.0);
```

Comment:
Piece of advice, in PHP never deal with floating value for prices.
It's not safe. better deal with cents (3 numbers after like 1000 to say 1€)
to understand why execute this in your console: 
```bash
$ php -a
php > var_dump(0.1 + 0.2 === 0.3);
```